src/parser.js

```diff
@@ -72,12 +72,12 @@
   }
```

Comment:
Just ignore the `src/parser.js`, as it's machine generated code (created through the `make parser` task).